hack/errata.py

```diff
@@ -83,13 +90,70 @@ def notify(message, webhook=None):
         print(message)
         return
 
+    msg_text = '<!subteam^STE7S7ZU2>: {fulladvisory} shipped {when}: {synopsis}'.format(**message)
+    if  message['approved_pr']:
+        msg_text += "\nPR {approved_pr} has been approved".format(**message)
+
     urllib.request.urlopen(webhook, data=urllib.parse.urlencode({
         'payload': {
-            'text': '<!subteam^STE7S7ZU2>: {fulladvisory} shipped {when}: {synopsis}'.format(**message),
+            'text': msg_text,
         },
     }).encode('utf-8'))
 
 
+def get_open_prs_to_fast(repo):
+    for pr in repo.get_pulls(state='open', sort='created', base='master'):
+        try:
+            # Skip unknown PRs
+            if not pr.title.startswith("Enable "):
+                continue
+            # Ignore PRs which don't target fast
+            if pr.title.split(" ")[3] != "fast":
+                continue
+            yield pr
+        except Exception as e:
+            print(f"Failed to parse {pr.number}: {e}")
+
+
+def extract_errata_number_from_body(body):
+    ERRATA_MARKER = 'https://errata.devel.redhat.com/advisory/'
+    first_line = body.split('\n')[0]
+    links = [
+        x for x in first_line.split(' ') if x.startswith(ERRATA_MARKER)
+    ]
+    if len(links) == 0:
+        print("No links found in PR body: {}".format(body))
```

Comment:
s/print/_LOGGER.warning/ for your diff.  Only `print` should be the no-webhook notification mode.

Reply:
Fixed in 8a1eb6a3401957bc0e83372ceb9ad673f851826a